pages.es/common/vim.md

```diff
@@ -1,7 +1,7 @@
 # vim
 
 > Vim (Vi IMproved), un editor de texto para la línea de comandos, que proporciona varios modos para diferentes tipos de manipulación de texto.
-> Pulsando `i` entra en el modo insertar. `<Esc>` habilita el uso de comandos de Vim.
+> Pulsando `i` entra en el modo insertar. `<Esc>` regresa al modo normal, lo que no permite insertar texto.
```

Comment:
I think the main page try to highlight which mode enables users to enter commands.
```suggestion
> Pulsando `i` entra en el modo insertar. `<Esc>` regresa al modo normal, permitiendo el uso de comandos Vim.
```